server/routes/index.js

```diff
@@ -1,10 +1,11 @@
 const express = require('express');
+const path = require('path');
 
 const router = express.Router();
 const voters = require('../model/voters');
 /* GET home page. */
 router.get('/', (req, res) => {
-  res.send('helloww');
+  res.sendFile(path.join(__dirname, '..', '..', 'client/build/index.html'));
```

Comment:
This is not correct. This path shouldn't be there. The only routes you need to have in your server are the ones that your client is going to query for data. React will render the home page and you will need to query the server for authentication, which should be set to a different route.

Reply:
we are not using react router, i understand that this approach is usually for setting up react router but we are not doing this.